test/TestHelper.js

```diff
@@ -74,4 +74,10 @@ export async function expectNoViolations(node, options = {}) {
   }
 
   expect(results.violations).to.be.empty;
-}
\ No newline at end of file
+}
+
+export function debounceFn(fn) {
```

Comment:
Proposal:

* expose `createDebounce(options = { flush: noop, cancel: noop })` and use it consistently.